test-scripts/tomcat-deployment/instance_map.csv

```diff
@@ -0,0 +1,2 @@
+1,Ubuntu14.04|JDK1.7,22233445566
+2,RedHaT7,JDK1.8,1524875
```

Comment:
Should this file end with a new line char? [1]

[1] https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
This is fixed with : 3b681e445a2e8d0047be71d8f83145453662dc71